itest/litd_firewall_test.go

```diff
@@ -891,6 +943,416 @@ func testSessionLinking(net *NetworkHarness, t *harnessTest) {
 	assertStatusErr(t.t, err, codes.ResourceExhausted)
 }
 
+// testChannelOpening tests that a client connected via a firewall can open
+// channels to a peer respecting the channel rules set by the firewall.
+func testChannelOpening(net *NetworkHarness, ht *harnessTest, t *testing.T) {
```

Comment:
Perhaps this test isn't the correct place for it, but I think we should add test coverage to prove that the `LndConnId` gets set to a different value if `litd` is restarted, i.e. with `net.RestartNode`.